src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+class ErrorBoundary extends React.Component<{}, { hasError: boolean }> {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
+
+  render() {
+    if (this.state.hasError) {
+      // TODO: render error banner
+      return <h1>Something went wrong.</h1>;
```

Comment:
Even if this is temporary, we should be log the error somewhere shouldn't we?

Reply:
for sure. we have not logged any bugs yet. i made a ticket to go round and log bugs based on severity. https://jira.mongodb.org/browse/EVG-7645